app/lib/bk/compat/parsers/circleci/orbs/docker/install_docker_credential_helper.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module BK
+  module Compat
+    module CircleCISteps
+      # Handles the translation of docker installation commands
+      class DockerOrb
+        def translate_docker_install_docker_credential_helper(config)
+          arch = config.fetch('arch', 'amd64')
+          helper_name = config.fetch('helper-name', '')
+          release_tag = config.fetch('release-tag', '')
+
+          [
+            '# Instead of installing docker credential helper in a step, ', \
+            '# we recommend your agent environment to have it pre-installed',
+            "echo '~~~ Installing Docker Credential Helper'",
+            'platform=""',
+            "arch=\"#{arch}\"",
+            "helper_name=\"#{helper_name}\"",
+            "release_tag=\"#{release_tag}\"",
+            'if [ "$(uname -s)" = "Darwin" ]; then',
+            '  platform="darwin"',
+            '  arch="arm64"',
+            'else',
+            '  platform="linux"',
+            "  arch=${arch}",
+            'fi',
+            "if [ -z \"${helper_name}\" ]; then",
+            '  if [ "${platform}" = "darwin" ]; then',
+            '    helper_name="osxkeychain"',
+            '  else',
+            '    helper_name="pass"',
+            '  fi',
+            'helper_filename="docker-credential-${helper_name}"',
+            'BIN_PATH="/usr/local/bin"',
+            'mkdir -p ${BIN_PATH}',
```

Comment:
In not other installation script that you created the _installation_ folder is created, why is it in this case?